gstreamer-plugin/gstsvthevcenc.c

```diff
@@ -731,19 +784,18 @@ set_default_svt_configuration (EB_H265_ENC_CONFIGURATION * svt_config)
   svt_config->encoderBitDepth = 8;
   svt_config->encoderColorFormat = EB_YUV420;
   svt_config->compressedTenBitFormat = FALSE;
-  svt_config->profile = 1;
+  svt_config->profile = 2;
   svt_config->tier = 0;
   svt_config->level = 0;
-  svt_config->injectorFrameRate =
-      (PROP_SPEEDCONTROL_DEFAULT > 0) ? PROP_SPEEDCONTROL_DEFAULT : 60 << 16;
-  svt_config->speedControlFlag = (PROP_SPEEDCONTROL_DEFAULT > 0);
-  svt_config->latencyMode = PROP_LOWLATENCY_DEFAULT;
+  svt_config->injectorFrameRate = 60 << 16;
```

Comment:
Does that require a shift operation? If so, I think comments should be added to the document and API header.

Reply:
There is a conversion of injectorFrameRate api internally. 
EbAppConfig.c, line 237:    
    cfg->injectorFrameRate = strtoul(value, NULL, 0);
    if (cfg->injectorFrameRate <= 1000 ){
        cfg->injectorFrameRate = cfg->injectorFrameRate << 16;
    }
later in EbTime.c
double                 injectorInterval  = (double)(1<<16)/(double)injectorFrameRate;

The injectorFrameRate and other APIs are for SvtHevcEncApp. The other implementation(ffmpeg, gst plugin) which use raw svt lib has to handle the api the same way with SvtHevcEncApp.

Reply:
I understand that config param in document (e.g. svt-hevc_encoder_user_guide) is for APP layer :)